Cargo.toml

```diff
@@ -13,8 +13,10 @@ publish = false
 crate-type = ["rlib", "staticlib"]
 
 [dependencies]
-filecoin-proofs = { git = "https://github.com/filecoin-project/rust-fil-proofs.git", branch = "master" }
-storage-proofs = { git = "https://github.com/filecoin-project/rust-fil-proofs.git", branch = "master" }
+# FIXME: This should be master branch or a version
```

Comment:
BLOCKING: To ensure I don't forget and merge the PR as-is, I've marked this PR as "Requesting Changes" until the TOML has been updated to point at rust-fil-proofs master. Once you've got rust-fil-proofs using your new merkle tree software and have updated the TOML, I'll approve the PR.

Reply:
fixed